File: Week_01/G20200343030533/LeetCode_026_533.py
"""
思路同C，只不过用Python写
"""
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums) == 0:
            return 0
        slow = 0
        fast = 0
        while fast < len(nums) :
            if nums[slow] != nums[fast]:
                slow += 1
                nums[slow] = nums[fast]
            fast += 1
        slow += 1
        return slow

File: Week_01/G20200343030533/LeetCode_026_533.java
//思路同C，只不过用java写
class Solution {
    public int removeDuplicates(int[] nums) {
        int j = 0;
        for (int i = 0; i < nums.length; i++){
            if (nums[i] != nums[j]){
                j++;
                nums[j] = nums[i];
            }
        }
        j++;
        return j;
        
    }
}
